cartridges/int_klaviyo_sfra/cartridge/scripts/utils/klaviyo/klaviyoDataLayer.js

```diff
@@ -82,24 +82,6 @@ var buildDataLayer = function () {
     return klData;
 };
 
-var buildCartDataLayer = function () {
```

Comment:
this does look like the core cartridge calls this, just checking to make sure that this doesn't need to be overriden?

Reply:
Both of the places that buildCartDataLayer are called are pointing to the function in the core klaviyoUtils file:
https://github.com/klaviyo/SFCC_Klaviyo/blob/67764a60e47a01d6591688405cef3e0bb93e169e/cartridges/int_klaviyo/cartridge/controllers/Klaviyo.js#L52
https://github.com/klaviyo/SFCC_Klaviyo/blob/67764a60e47a01d6591688405cef3e0bb93e169e/cartridges/int_klaviyo_sfra/cartridge/controllers/Klaviyo.js#L54